ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -509,7 +509,7 @@ tests:
   steps:
     cluster_profile: nutanix
     workflow: openshift-e2e-nutanix
-- as: e2e-agent-compact
+- as: e2e-agent-compact-ipv4
```

Comment:
Based on our discussed testing matrix, shouldn't the mce deployment be moved into a periodic one?

Reply:
Yes but I think we should address that in a separate PR as this just adds the dualstack job (the "ipv4" was added to the agent-compact job based on the earlier comment)